bin/bashrc

```diff
@@ -1,4 +1,7 @@
-## ~/.bashrc file portion for 'desiproc' user @ NERSC.
```

Comment:
I propose we rename this file something like `setup-legacypipe` or `legacypipe-envs` or something much more creative and appropriate.  Then the documentation just says that this file needs to be sourced.